# Translations template for ceilometer.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the ceilometer project.
#
# Translators:
# Seong-ho Cho <darkcircle.0426@gmail.com>, 2014
# Seunghyo Chun <seunghyo.chun@gmail.com>, 2013
# Seunghyo Chun <seunghyo.chun@gmail.com>, 2013
# Sungjin Kang <potopro@gmail.com>, 2013
# Sungjin Kang <potopro@gmail.com>, 2013
# Lucas Palm <lapalm@us.ibm.com>, 2015. #zanata
# OpenStack Infra <zanata@openstack.org>, 2015. #zanata
# Lucas Palm <lapalm@us.ibm.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: ceilometer 6.0.0.0b4.dev6\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2016-03-04 20:29+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2016-02-03 05:50+0000\n"
"Last-Translator: Lucas Palm <lapalm@us.ibm.com>\n"
"Language: ko-KR\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Korean (South Korea)\n"

#, python-format
msgid "%(entity)s %(id)s Not Found"
msgstr "%(entity)s %(id)s을(를) 찾을 수 없음"

#, python-format
msgid "Arithmetic transformer must use at least one meter in expression '%s'"
msgstr "'%s' 표현식에서 산술 변환기는 하나 이상의 미터를 사용해야 함"

#, python-format
msgid "Cannot create table %(table_name)s   it already exists. Ignoring error"
msgstr "%(table_name)s 테이블을 작성할 수 없음, 이미 존재합니다. 오류 무시"

#, python-format
msgid "Continue after error from %(name)s: %(error)s"
msgstr "%(name)s에서 오류 후 계속: %(error)s"

#, python-format
msgid "Could not connect to XenAPI: %s"
msgstr "XenAPI를 연결할 수 없음: %s"

#, python-format
msgid "Could not get CPU Util for %(id)s: %(e)s"
msgstr "%(id)s에 대해 CPU Util을 가져올 수 없음: %(e)s"

#, python-format
msgid "Could not get Memory Usage for %(id)s: %(e)s"
msgstr "%(id)s에 대한 메모리 사용을 가져올 수 없음: %(e)s"

#, python-format
msgid "Could not get VM %s CPU Utilization"
msgstr "VM %s CPU 이용률을 가져올 수 없음"

#, python-format
msgid "Couldn't obtain IP address of instance %s"
msgstr "%s 인스턴스의 IP 주소를 얻을 수 없음"

msgid ""
"Dispatcher target was not set, no meter will be posted. Set the target in "
"the ceilometer.conf file"
msgstr ""
"디스패처 대상을 설정하지 않았으며 미터가 게시되지 않습니다. ceilometer.conf "
"파일에 대상을 설정하십시오."

#, python-format
msgid "Dropping Notification %(type)s (uuid:%(msgid)s)"
msgstr "알림 %(type)s 삭제 중(uuid:%(msgid)s)"

#, python-format
msgid ""
"Error from libvirt while looking up instance <name=%(name)s, id=%(id)s>: "
"[Error Code %(error_code)s] %(ex)s"
msgstr ""
"인스턴스 <name=%(name)s, id=%(id)s> 검색 중 libvirt에서 오류 발생: [오류 코"
"드 %(error_code)s] %(ex)s"

#, python-format
msgid "Error parsing HTTP response: %s"
msgstr "HTTP 응답 구문 분석 중 오류 발생: %s"

msgid "Error stopping pollster."
msgstr "의견조사자를 중지하는 중에 오류가 발생했습니다. "

msgid "Event"
msgstr "이벤트"

msgid "Expression evaluated to a NaN value!"
msgstr "표현식이 NaN 값으로 평가되었습니다!"

#, python-format
msgid "Failed to import extension for %(name)s: %(error)s"
msgstr "%(name)s 확장자를 가져오는 데 실패함: %(error)s"

#, python-format
msgid ""
"Failed to inspect data of instance <name=%(name)s, id=%(id)s>, domain state "
"is SHUTOFF."
msgstr ""
"인스턴스 <이름=%(name)s, id=%(id)s>의 데이터 검사 실패, 도메인 상태가 SHUTOFF"
"입니다."

#, python-format
msgid ""
"Failed to inspect memory usage of %(instance_uuid)s, can not get info from "
"libvirt: %(error)s"
msgstr ""
"%(instance_uuid)s의 메모리 사용량 검사 실패, libvirt에서 정보를 가져올 수 없"
"음: %(error)s"

#, python-format
msgid ""
"Failed to inspect memory usage of instance <name=%(name)s, id=%(id)s>, can "
"not get info from libvirt."
msgstr ""
"인스턴스 <이름=%(name)s, id=%(id)s>의 메모리 사용량 검사 실패, libvirt에서 정"
"보를 가져올 수 없습니다."

#, python-format
msgid "Failed to load any notification handlers for %s"
msgstr "%s의 알림 핸들러 로드 실패"

#, python-format
msgid "Failed to parse the timestamp value %s"
msgstr "시간소인 값 %s 구문 분석 실패"

#, python-format
msgid "Failed to publish %d datapoints, dropping them"
msgstr "%d 데이터포인트 공개 실패. 이를 삭제하는 중"

#, python-format
msgid "Failed to publish %d datapoints, queue them"
msgstr "%d 데이터포인트 공개 실패. 이를 큐에 대기시킴"

#, python-format
msgid "Failed to record metering data: %s"
msgstr "측정 데이터 기록 실패: %s"

#, python-format
msgid "Filter expression not valid: %s"
msgstr "필터 표현식이 올바르지 않음: %s"

#, python-format
msgid "Ignoring instance %(name)s (%(instance_id)s) : %(error)s"
msgstr "인스턴스 %(name)s (%(instance_id)s) 무시 중: %(error)s"

#, python-format
msgid "Ignoring instance %(name)s: %(error)s"
msgstr "인스턴스 %(name)s 무시 중: %(error)s"

#, python-format
msgid "Invalid period %(period)s: %(err)s"
msgstr "올바르지 않은 기간 %(period)s: %(err)s"

#, python-format
msgid "Invalid trait type '%(type)s' for trait %(trait)s"
msgstr "특성 %(trait)s에 대한 올바르지 않은 특성 유형 '%(type)s'"

msgid "Limit must be positive"
msgstr "제한 값은 양수여야 합니다."

#, python-format
msgid "More than one event with id %s returned from storage driver"
msgstr "ID가 %s인 둘 이상의 이벤트가 스토리지 드라이버에서 리턴됨"

#, python-format
msgid "Multiple VM %s found in XenServer"
msgstr "여러 VM %s을(를) XenServer에서 찾음 "

msgid "Must specify connection_url, and connection_password to use"
msgstr "사용할 connection_url 및 connection_password를 지정해야 함 "

msgid "Node Manager init failed"
msgstr "노드 관리자 초기화 실패"

#, python-format
msgid "Not Authorized to access %(aspect)s %(id)s"
msgstr "%(aspect)s %(id)s에 대한 액세스 권한이 부여되지 않음"

#, python-format
msgid "OpenDaylitght API returned %(status)s %(reason)s"
msgstr "OpenDaylitght API가 %(status)s 리턴: %(reason)s"

#, python-format
msgid "Opencontrail API returned %(status)s %(reason)s"
msgstr "Opencontrail API가 %(status)s 리턴: %(reason)s"

#, python-format
msgid "Order-by expression not valid: %s"
msgstr "Order-by 표현식이 올바르지 않음: %s"

msgid "Period must be positive."
msgstr "기간은 양수여야 합니다. "

#, python-format
msgid "Pipeline %(pipeline)s: %(status)s after error from publisher %(pub)s"
msgstr "파이프라인 %(pipeline)s: 공개자 %(pub)s에서 오류 후 %(status)s"

#, python-format
msgid "Pipeline %(pipeline)s: Continue after error from publisher %(pub)s"
msgstr "파이프라인 %(pipeline)s: 공개자 %(pub)s에서 오류 후 계속"

#, python-format
msgid "Pipeline %(pipeline)s: Error flushing transformer %(trans)s"
msgstr "파이프라인 %(pipeline)s: 변환기 %(trans)s을(를) 비우는 중 오류 발생"

#, python-format
msgid ""
"Pipeline %(pipeline)s: Exit after error from transformer %(trans)s for "
"%(smp)s"
msgstr "파이프라인 %(pipeline)s: %(smp)s의 변환기 %(trans)s에서 오류 후 종료"

#, python-format
msgid "Polling %(mtr)s sensor failed for %(cnt)s times!"
msgstr "폴링 %(mtr)s 센서가 %(cnt)s번 실패했습니다!"

#, python-format
msgid "Pollster for %s is disabled!"
msgstr "%s의 의견조사자가 사용 안함으로 설정되어 있습니다!"

#, python-format
msgid "Prevent pollster %(name)s for polling source %(source)s anymore!"
msgstr ""
"의견조사자 %(name)s이(가) 소스 %(source)s를 더 이상 폴링하지 않도록 하십시오!"

#, python-format
msgid ""
"Publisher max local_queue length is exceeded, dropping %d oldest samples"
msgstr "공개자 최대 local_queue 길이가 초과됨. %d 가장 오래된 샘플 삭제 중"

#, python-format
msgid "Publishing policy is unknown (%s) force to default"
msgstr "공개 정책을 알 수 없음(%s). 기본값으로 강제 설정함"

#, python-format
msgid "RGW AdminOps API returned %(status)s %(reason)s"
msgstr "RGW AdminOps API가 %(status)s %(reason)s을(를) 리턴함"

msgid "Request failed to connect to OpenDaylight with NorthBound REST API"
msgstr "요청이 NorthBound REST API로 OpenDaylight에 연결하는 데 실패함"

#, python-format
msgid "Required field %s not specified"
msgstr "필수 필드 %s이(가) 지정되지 않음"

msgid "Resource"
msgstr "리소스"

msgid "Sample"
msgstr "샘플"

msgid "Samples should be included in request body"
msgstr "샘플이 요청 본문에 포함되어야 함"

#, python-format
msgid "Skip loading extension for %s"
msgstr "%s 확장자 로드 건너뛰기"

#, python-format
msgid "String %s is not a valid isotime"
msgstr "문자열 %s이(가) 올바른 등시간이 아님"

msgid ""
"The Yaml file that defines mapping between samples and gnocchi resources/"
"metrics"
msgstr "샘플과 gnocchi resources/ 메트릭 간 맵핑을 정의하는 Yaml 파일"

#, python-format
msgid ""
"The data type %(type)s is not supported. The supported data type list is: "
"%(supported)s"
msgstr ""
"데이터 유형 %(type)s이(가) 지원되지 않습니다. 지원되는 데이터 유형 목록은 "
"%(supported)s입니다."

msgid "The path for the file publisher is required"
msgstr "파일 공개자의 경로가 필요함"

#, python-format
msgid "UDP: Cannot decode data sent by %s"
msgstr " UDP: %s이(가) 보낸 데이터를 해독할 수 없습니다"

msgid "UDP: Unable to store meter"
msgstr "UDP: 측정을 저장할 수 없습니다"

#, python-format
msgid "Unable to connect to the database server: %(errmsg)s."
msgstr "데이터베이스 서버에 연결할 수 없음: %(errmsg)s."

#, python-format
msgid ""
"Unable to convert the value %(value)s to the expected data type %(type)s."
msgstr "%(value)s 값을 예상 데이터 유형 %(type)s(으)로 변환할 수 없습니다."

#, python-format
msgid "Unable to discover resources: %s"
msgstr "자원을 검색할 수 없음: %s"

#, python-format
msgid "Unable to evaluate expression %(expr)s: %(exc)s"
msgstr "%(expr)s 표현식을 평가할 수 없음: %(exc)s"

#, python-format
msgid "Unable to load publisher %s"
msgstr "공개자 %s을(를) 로드할 수 없음"

#, python-format
msgid "Unable to load the hypervisor inspector: %s"
msgstr "하이퍼바이저 검사기를 로드할 수 없음: %s"

#, python-format
msgid ""
"Unable to reconnect to the primary mongodb after %(retries)d retries. Giving "
"up."
msgstr ""
"%(retries)d회 재시도한 이후에는 1차 mongodb에 다시 연결할 수 없습니다. 포기하"
"는 중입니다."

#, python-format
msgid ""
"Unable to reconnect to the primary mongodb: %(errmsg)s. Trying again in "
"%(retry_interval)d seconds."
msgstr ""
"1차 mongodb에 다시 연결할 수 없음: %(errmsg)s. %(retry_interval)d초 후에 다"
"시 시도합니다."

msgid "Unable to send sample over UDP"
msgstr "UDP를 통해 샘플을 전송할 수 없음"

#, python-format
msgid ""
"Unexpected exception converting %(value)s to the expected data type %(type)s."
msgstr ""
"%(value)s을(를) 예상된 데이터 유형으로 변환하는 중에 예상치 않은 예외 발생 "
"%(type)s."

#, python-format
msgid "Unknown discovery extension: %s"
msgstr "알 수 없는 검색 확장자: %s"

#, python-format
msgid "Unknown metadata type. Key (%s) will not be queryable."
msgstr "알 수 없는 메타데이터 유형입니다. 키(%s)를 조회할 수 없습니다."

#, python-format
msgid "Unknown status %(stat)s received on fw %(id)s,skipping sample"
msgstr ""
"fw %(id)s에서 알 수 없는 상태 %(stat)s이(가) 수신됨. 샘플을 건너뛰는 중"

#, python-format
msgid "Unknown status %(stat)s received on pool %(id)s, skipping sample"
msgstr ""
"풀 %(id)s에서 알 수 없는 상태 %(stat)s이(가) 수신됨. 샘플을 건너뛰는 중"

#, python-format
msgid "Unknown status %(stat)s received on vip %(id)s, skipping sample"
msgstr ""
"vip %(id)s에서 알 수 없는 상태 %(stat)s이(가) 수신됨. 샘플을 건너뛰는 중"

#, python-format
msgid "VM %s not found in VMware vSphere"
msgstr "VM %s을(를) VMware vSphere에서 찾을 수 없음"

#, python-format
msgid "VM %s not found in XenServer"
msgstr "VM %s을(를) XenServer에서 찾을 수 없음 "

msgid "Wrong sensor type"
msgstr "잘못된 센서 유형"

msgid "XenAPI not installed"
msgstr "XenAPI가 설치되지 않음"

#, python-format
msgid "could not get CPU time for %(id)s: %(e)s"
msgstr "%(id)s의 CPU 시간을 가져올 수 없음: %(e)s"

#, python-format
msgid "dropping out of time order sample: %s"
msgstr "시간 순서 샘플에서 벗어남: %s"

#, python-format
msgid "dropping sample with no predecessor: %s"
msgstr "선행 작업이 없는 샘플 삭제: %s"

msgid "ipmitool output length mismatch"
msgstr "ipmitool 출력 길이 불일치"

msgid "max_bytes and backup_count should be numbers."
msgstr "max_bytes 및 backup_count는 숫자여야 합니다."

#, python-format
msgid "message signature invalid, discarding message: %r"
msgstr "올바르지 않은 메시지 서명. 메시지 버리는 중: %r"

msgid "parse IPMI sensor data failed,No data retrieved from given input"
msgstr ""
"IPMI 센서 데이터 구문 분석에 실패했음, 제공된 입력에서 검색된 데이터가 없음"

msgid "parse IPMI sensor data failed,unknown sensor type"
msgstr "IPMI 센서 데이터 구문 분석에 실패했음, 알 수 없는 센서 유형"

msgid "running ipmitool failure"
msgstr "ipmitool 실행 실패"
